ui/app/components/ui/searchable-item-list/searchable-item-list.js

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from 'react'
+import PropTypes from 'prop-types'
+import ItemList from './item-list'
+import ListItemSearch from './list-item-search'
+
+export default function SearchableItemList ({
+  itemSelectorError = '',
```

Comment:
Most of these default values are pointless (i.e. things work identically without them).

I'd strongly prefer that we reduce the complexity where possible by getting rid of these branches that don't have any effect.

Reply:
`ItemList` has a good few useless defaults as well